projects/gameboard-ui/src/app/game/gameboard-page/gameboard-page.component.ts

```diff
@@ -205,58 +207,8 @@ export class GameboardPageComponent implements OnDestroy {
   mouseenter(e: MouseEvent, spec: BoardSpec) {
     this.hoveredItem = spec;
     spec.c = 'purple';
```

Comment:
if cleaning this up, remove the related properties as well (L23-26) and the renderer2 constructor param.